rhobs/alerting/data_plane/prometheus.pipeline_to_snapshot_alerts.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-pipeline-to-snapshot-exceeded-alerting-rules
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: pipeline-to-snapshot-exceeded
+    interval: 1m
+    rules:
+    - alert: PipelineRunToSnapshotExceeded
+      expr: |
+       (
+        (
+         sum(increase(integration_svc_response_seconds_bucket{le="+Inf"}[5m])) by (namespace, source_cluster) -
+         sum(increase(integration_svc_response_seconds_bucket{le="30"}[5m])) by (namespace, source_cluster)
+        ) /
+        sum(increase(integration_svc_response_seconds_bucket{le="+Inf"}[5m])) by (namespace, source_cluster)
+       ) > 0.10
+      for: 1m
+      labels:
+        severity: warning
+        slo: true
+      annotations:
+        summary: >-
+          PipelineRunFinish to SnapshotInProgress time exceeded
+        description: >-
+          Time from pipeline run finished to snapshot marked in progress has been over 30s for more than 10% of requests during the last 5 minutes on cluster {{ $labels.source_cluster }}
```

Let's keep the line length less than 90 chars. You should be able to do the same on the expected results, if you start there with `>-` as well.
```suggestion
        description: >-
          Time from pipeline run finished to snapshot marked in progress has been over
          30s for more than 10% of requests during the last 5 minutes on cluster
          {{ $labels.source_cluster }}
```

---

done.